.github/workflows/static_checks.yml

```diff
@@ -78,17 +78,28 @@ jobs:
       # For more information on shellcheck failures:
       # https://github.com/koalaman/shellcheck/wiki/Checks
       run: |
-        find ./bin -type f \
-        ! -name docker-build-proxy \
-        ! -name _docker.sh \
-        ! -name fmt \
-        ! -name lint \
-        ! -name _log.sh \
-        ! -name minikube-start-hyperv.bat \
-        ! -name _tag.sh \
-        ! -name test-cleanup \
-        ! -name _test-run.sh \
-        ! -name update-go-deps-shas \
-        ! -name *.nuspec \
-        ! -name *.ps1 \
-        | xargs -I {} bin/shellcheck -x -P ./bin {}
+        bin/shellcheck -x -P ./bin $(find . -type f \
```

Comment:
@joakimr-axis What are you thoughts on adding this entire command to its own `bin/shellcheck-all` (or something similar to that name?).

We have a few scripts like this already, where `bin/kind` or `bin/helm` exist, but then there is a helper script that does more with that command (e.g. `bin/helm-build`).

That way, this job could just refer to that script *and* users could run it themselves. Currently, the only way to shellcheck the entire project before pushing a change would be to copy-paste this section of the job.

Reply:
I like that!

Reply:
Done